doc/engineering-reference/src/simulation-models-encyclopedic-reference-003/zone-internal-gains.tex

```diff
@@ -8,6 +8,78 @@ \subsection{Heat Gain from Lights}\label{heat-gain-from-lights}
 
 The input object Lights provides a model for internal gains from lights.~ Radiant gains from lights must be handled differently from other radiant gains for reasons described here (long wavelength description).~ The total radiant gains from lights must be divided into visible and thermal portions.~ For example, the total electric input to typical incandescent lights is converted to 10\% visible radiation, 80\% thermal radiation, and 10\% convective gain.~ In contrast, the electric input to typical fluorescent lights is converted to 20\% visible radiation, 20\% thermal radiation, and 60\% convective gain (see Carrier 1965).~ These percentage splits are under user control with the Lights input object.
 
+When the value of Return Air Fraction is greater than 0 and  the return air flow rate is greater than 0, the portion of light heat will be added to the return air node. The added heat will make the node temperature rise. The temperature difference is calculated in the CalcZoneLeavingConditions funtion of the ZoneEquipmentManager module:
+
+
+\begin{equation}
+TempRetRise =  \frac{QRetAir} {MassFlowRA * CpAir}
+\end{equation}
+
+where:
+
+\(TempRetRise\) is the temperature rise due to added light heat gain (C)
+
+\(QRetAir\) is the added heat gain from return air fraction (W)
+
+\(MassFlowRA\) is the mass flow rate at the return node (kg/s)
+
+\(CpAir\) is the specific heat (J/kg)
+
+When both node names of return air and exhaust air are entered in the Lights object, the return heat is shared by both nodes. Both nodes will have the same temperature due to added return heat:
+
+\begin{equation}
+TempRetRise = \frac{QRetAir} {(MassFlowRA + MassFlowEx ) * CpAir}
+\end{equation}
+
+where:
+
+\(TempRetRise\) is the temperature rise due to added light heat gain (C)
+
+\(QRetAir\) is the added heat gain from return air fraction (W)
+
+\(MassFlowRA\) is the mass flow rate at the return node (kg/s)
+
+\(MassFlowEx\) is the mass flow rate at the exhasut node (kg/s)
+
+\(CpAir\) is the specific heat (J/kg)
+
+Although a single Lights object allows a return node and an exhasut node, it is possible to have multiple Lights objects with different return nodes and referrred to the same exhaust node. Therefore, the exhaust node will share return heat from different return nodes. The temperature rise at i-th return node is given below:
```

Comment:
Typo: exhasut

This is not the scenario I was asking about. What would this mean physically for a single zonehvac unit to draw from 2 different return ducts? I'm not sure we should allow this combination.

I was asking about the reverse, where 2 different zoneHVAC units draw from the same return duct. At this point, I'm ok with not allowing that just so we can call this complete.

Reply:
@mjwitte Thanks for finding typos. I corrected 4. I am going to upload the Eng. Ref.

I can remove the scenario for a single zonehvac unit to draw from 2 different return ducts. Please let me know if the scenario is removed or not.